Makefile

```diff
@@ -8,6 +8,6 @@ gen: solidity-wrappers
 
 SOLIDITY_DIR = .
 SOLIDITY_SRC_DIR = $(SOLIDITY_DIR)/src
-CONTRACTS = QuantumGravityBridge.sol
+CONTRACTS = QuantumGravityBridge.sol DAVerifier.sol
```

Comment:
The DAVerifier wrapper isn't checked in CI, only the QuantumGravityBridge wrapper. Can the former be added?

Reply:
[61f844b](https://github.com/celestiaorg/quantum-gravity-bridge/pull/138/commits/61f844b88328446e0f098bec12a2296bc5fcc7f4)